src/tests/test_process.py

```diff
@@ -94,8 +129,35 @@ def _test_threads(self):
         #   - InvBroadcaster
         #   - AddrBroadcaster
         #   - Downloader
-        self.assertEqual(
-            len(self.process.threads()), self._threads_count)
+        #   - Uploader
+
+        self.longMessage = True
+
```

Comment:
If this hunk is platform dependent, maybe here should be some condition?

Reply:
I haven't tested it on Windows, but I assumed the result will be empty. Maybe it can be wrapped in try/except. The output is only used if there is an assertion error, it's not an assertion itself.

Reply:
https://travis-ci.org/github/g1itch/PyBitmessage/jobs/738510444